docs/source/aliases.md

```diff
@@ -0,0 +1,132 @@
+---
+html_meta:
+  "description": "Aliases - a mechanism to redirect old URLs to new ones."
```

Comment:
Any particular reason why we call this "aliases" instead of "redirects"?

Reply:
@tisto I think it started from Plone Classic, where the view on context is called `@@manage-aliases`

I don't have a strong opinion on it, but I think I like more `aliases` than `redirects`.

Reply:
I don't have a strong opinion here either. Maybe others want to weigh in here. We should just be consistent in Plone Classic and Volto.